portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,26 +33,37 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-  ArrayList<String> messageArrayList = new ArrayList<String>();
+  ArrayList<Comment> commentArrayList = new ArrayList<Comment>();
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Query commentQuery = new Query("Comment").addSort("timestamp", SortDirection.DESCENDING);
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    PreparedQuery commentResults = datastore.prepare(commentQuery);
+    for (Entity comment : commentResults.asIterable()) {
+        String comment_text = (String) comment.getProperty("comment");
```

Comment:
For comment_text and comment_object, please use camelcase to conform to Java style guide. So they would become commentText and commentObject

Reply:
Updated variable name.